pwiz_tools/Skyline/Test/LongPathDirectoryTest.cs

```diff
@@ -129,13 +129,13 @@ public void DirectoryWithLongPathTest()
             if (IsVerboseMode)
                 Console.WriteLine($@"Creating directory with path ""{inputPath}"" that has length {inputPath.Length} characters");
 
-            DirectoryEx.CreateLongPath(inputPath);
+            Directory.CreateDirectory(inputPath);
```

Comment:
Thinking about this more. I think you should leave the testing of DirectoryEx and add this as extra testing which is great confirmation that the config solution works. However, since this PR does not remove the code in DirectoryEx. It still needs to get tested until it is removed, if it is removed. That will be a separate PR.